src/api/archive_unstable_storage.md

```diff
@@ -3,91 +3,119 @@
 **Parameters**:
 
 - `hash`: String containing a hexadecimal-encoded hash of the header of the block whose storage to fetch.
-- `key`: String containing the hexadecimal-encoded key to fetch in the storage.
+- `items`: Array of objects. The structure of these objects is found below.
 - `childTrie`: `null` for main storage look-ups, or a string containing the hexadecimal-encoded key of the child trie of the "default" namespace.
-- `includeDescendants`: Boolean indicating whether the key-values of all the descendants of the `key` should be returned as well.
 
-**Return value**: String containing an opaque value representing the operation, or `null` if no block with that `hash` exists.
+Each element in `items` must be an object containing the following fields:
+
+- `key`: String containing the hexadecimal-encoded key to fetch in the storage.
+- `type`: String equal to one of: `value`, `hash`, `closestDescendantMerkleValue`, `descendantsValues`, `descendantsHashes`.
+- `paginationStartKey`: This parameter is optional and should be a string containing the hexadecimal-encoded key from which the storage iteration should resume. This parameter is only valid in the context of `descendantsValues` and `descendantsHashes`.
+
+**Note**:
```

Comment:
For consistency with chainHead_storage, I think that this Note section should be moved to be right above the Possible Errors section.

(Also, I'd want to see the return value near the input args rather than below a bunch of text :))

Reply:
(and maybe rename it to "## Overview" too)